browser/js/common/directives/snippet/snippet.html

```diff
@@ -15,7 +15,7 @@
             <span ng-click="removeCollaborator(collaborator.$id)" class="close"><i class="material-icons"> clear</i></span>
 
             <md-tooltip>{{collaborator.first_name}} {{collaborator.last_name}}</md-tooltip>
-            <img ng-src="{{ collaborator.photoUrl || '/files/default-profile.png' }}" style="z-index: {{ collaborators.length - index }}">
+            <img ng-src="{{ collaborator.photoUrl || '/files/default-profile.png' }}" style="z-index: {{ collaborators.length - index }}" ng-cloak>
```

Comment:
You might consider creating users instances with a default `photoUrl`—like stored in the db.